.circleci/config.yml

```diff
@@ -0,0 +1,20 @@
+version: 2
+
+jobs:
+  build:
+    docker:
+      - image: circleci/python:2.7.15-jessie-browsers
```

Comment:
I think we can just use `circleci/python:2.7`, from [documentation](https://hub.docker.com/r/circleci/python/):

> `python:<version>`
> This is the defacto image. If you are unsure about what your needs are, you probably want to use this one for common tools.

> `python:<version>-browsers`
> This image makes browser testing easier. It extends the defacto image and installs Firefox and Chrome/chromedriver and configure them to run in a chrontainzied environment